code/hud/hud.cpp

```diff
@@ -3049,7 +3049,19 @@ void hud_set_dim_color()
  * @param objp			Object to test for team color to base on
```

Comment:
Does this documentation still make sense?

Reply:
I had thought about that, too. The object affects what the team color will be set to, so I left it as is. The alternative I had tried was `Object to test for team color to set`, but that didn't really sound right, as it's not setting a team color for the object.

Reply:
Updated with discussion from Discord to clarify comments.